ajax/libs/sketchpad/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "sketchpad",
+  "filename": "scripts/sketchpad.min.js",
+  "version": "0.1.0",
+  "description": "A simple sketchpad.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yiom/sketchpad.git"
+  },
+  "homepage": "https://yiom.github.io/sketchpad/",
+  "keywords": [
+    "sketchpad",
+    "javascript",
+    "svg"
+  ],
+  "authors": [
+    "Nihey Takizawa <nihey.takizawa@gmail.com>",
+    "Jean Lucas <jeanleonino@gmail.com>",
+    "Yiom"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/yiom/sketchpad.git",
+    "fileMap": [
```

Comment:
Because the files which under `scripts` & `styles` are using `""` as basePath.
Let's revise as the followings:
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/yiom/sketchpad.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "+(scripts|styles)/+(initialize|sketchpad)*"
        ]
      }
    ]
  }
```

Many thanks.